mifosng-android/src/main/java/com/mifos/api/local/MifosSQLCipherHelper.java

```diff
@@ -0,0 +1,18 @@
+package com.mifos.api.local;
+
+import com.raizlabs.android.dbflow.config.DatabaseDefinition;
+import com.raizlabs.android.dbflow.structure.database.DatabaseHelperListener;
+import com.raizlabs.dbflow.android.sqlcipher.SQLCipherOpenHelper;
+
+public class MifosSQLCipherHelper extends SQLCipherOpenHelper {
+
+    public MifosSQLCipherHelper(DatabaseDefinition databaseDefinition,
+                                DatabaseHelperListener listener) {
+        super(databaseDefinition, listener);
+    }
+
+    @Override
+    protected String getCipherSecret() {
+        return "dbflow-rules";
```

Comment:
I don't know how this CipherSecret works in DBFlow but don't you think we should generate it differently for each App Install.

Let's say we have a function that takes certain inputs and returns an output. For a given input the output should always be the same. 

Here is how it would look

SOME VALUE XYZ which is unique per device (Android user) ----> Function -----> Output is some hash

use this Hash as a Cipher Secret. 

So that it becomes even more difficult to decode the encrypted DB. 

Makes sense ?